src/main/java/edu/pw/eiti/pik/base/event/ParticipationForUserCreationEvent.java

```diff
@@ -0,0 +1,13 @@
+package edu.pw.eiti.pik.base.event;
+
+import edu.pw.eiti.pik.participation.Participation;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class ParticipationForUserCreationEvent {
```

Comment:
This event can be called simply ParticipationCreation - it will be more generic and reusable in next tasks.